NEWS

```diff
@@ -1 +0,0 @@
-See docs/news.rst
```

Comment:
I’m not against this change, but since the file was added [on purpose](https://github.com/scrapy/scrapy/commit/f5b87dbef8f3c657e31eeb026d8a77b0ccc77c29) and nothing has changed since then, I’m not sure why we should remove it now.